streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/TaskAssignorConvergenceTest.java

```diff
@@ -236,16 +233,17 @@ public void staticAssignmentShouldConvergeWithTheFirstAssignment() {
                                                                 0,
                                                                 1000L);
 
-        final Harness harness = Harness.initializeCluster(1, 1, 1);
+        final Harness harness = Harness.initializeCluster(1, 1, 1, () -> 1);
 
         testForConvergence(harness, configs, 1);
         verifyValidAssignment(0, harness);
+        verifyBalancedAssignment(harness);
     }
 
     @Test
     public void assignmentShouldConvergeAfterAddingNode() {
-        final int numStatelessTasks = 15;
-        final int numStatefulTasks = 13;
+        final int numStatelessTasks = 7;
```

Comment:
Not sure what I was thinking before. What I wanted was for all the key numbers to be coprime. Of course, the easiest way to do this is to only use prime numbers, but you might notice that 15 is not prime ;) Now it's fixed.

Reply:
Well, if you have a set of N prime numbers and one number which isn't, aren't they all still coprime? :P

Reply:
Right, but I think there's a "3" or a "5" in there somewhere, maybe in the other test. Anyway, my _intent_ was to make them all prime so I wouldn't have to think to hard about whether they were all coprime. But, in reality, I managed to screw up both.